bin/test-browser.js

```diff
@@ -16,7 +16,7 @@ var testTimeout = 30 * 60 * 1000;
 var username = process.env.SAUCE_USERNAME;
 var accessKey = process.env.SAUCE_ACCESS_KEY;
 
-var SELENIUM_VERSION = process.env.SELENIUM_VERSION || '2.46.0';
+var SELENIUM_VERSION = process.env.SELENIUM_VERSION || '2.47.1';
```

Comment:
the selenium upgrade was not necessary if we are specifying the firefox version, but I figured why not